src/reference/00-Getting-Started/02-sbt-by-example.md

```diff
@@ -541,13 +541,14 @@ or a continuous test like `~testQuick`.
 You can use the sbt `new` command to quickly setup a simple "Hello world" build.
 
 ```
-\$ sbt new scala/scala-seed.g8
-....
-A minimal Scala project.
+\$ sbt new scala/scala3.g8
```

Comment:
scala/toolkit.local also defaults to Scala 3:

```bash
Select a template: a
Scala version (default: 3.3.0):
Scala Toolkit version (default: 0.2.0):
```

Reply:
Added a commit with the adjustment I think you're suggesting.